src/components/constants.ts

```diff
@@ -53,6 +53,7 @@ export const ACTION_CATEGORIES: ActionCategories = {
     { name: 'evolution', label: 'Compute evolution' },
     { name: 'cumsum', label: 'Cumulated sum' },
     { name: 'percentage', label: 'Percentage of total' },
+    { name: 'statistics', label: 'Statistics' },
```

Comment:
@rhuille the step is still not accessible from the "Other operations" of the columns menu. I think it would be nice though and does not cost much. What do you think?